aequilibrae/utils/create_delaunay_network.py

```diff
@@ -0,0 +1,124 @@
+        edges = edges.assign(direction=0, distance=0, link_id=np.arange(edges.shape[0]) + 1)
+        edges = edges[['link_id', 'direction', 'a_node', 'b_node', 'distance']]
+        edges.to_sql(DELAUNAY_TABLE, conn, index=False)
+
+        # Now we create the geometries for the delaunay triangulation
+        conn.execute("select AddGeometryColumn( 'delaunay_network', 'geometry', 4326, 'LINESTRING', 'XY', 0);")
+        conn.execute("CREATE UNIQUE INDEX unique_link_id_delaunay on delaunay_network(link_id)")
+
+        node_geo_sql = '''INSERT INTO delaunay_network (link_id, geometry)
+                             select lnk.link_id, MakeLine(nd.geometry, nf.geometry) from delaunay_network lnk
+                                    inner join nodes nd on lnk.a_node=nd.node_id
+                                    inner join nodes nf on lnk.b_node=nf.node_id
+                           ON CONFLICT(link_id) DO UPDATE SET geometry=excluded.geometry'''
+
+        zone_geo_sql = '''INSERT INTO delaunay_network (link_id, geometry)
+                             select lnk.link_id, MakeLine(st_centroid(za.geometry), st_centroid(zb.geometry)) from delaunay_network lnk
+                                    inner join zones za on lnk.a_node=za.zone_id
+                                    inner join zones zb on lnk.b_node=zb.zone_id
+                          ON CONFLICT(link_id) DO UPDATE SET geometry=excluded.geometry;'''
+
+        sql = zone_geo_sql if source == 'zones' else node_geo_sql
+        conn.execute(sql)
+        # Updates link distance
+        conn.execute('update delaunay_network set distance=GeodesicLength(geometry);')
+        conn.commit()
+        conn.close()
+
+    def assign_matrix(self, matrix: AequilibraeMatrix, result_name: str):
+        conn = database_connection()
+
+        sql = f"select link_id, direction, a_node, b_node, distance, 1 capacity from {DELAUNAY_TABLE}"
+
+        df = pd.read_sql(sql, conn)
+        centroids = np.array(np.unique(np.hstack((df.a_node.values, df.b_node.values))), int)
+
+        g = Graph()
+        g.mode = 'delaunay'
+        g.network = df
+        g.prepare_graph(centroids)
+        g.set_blocked_centroid_flows(True)
+
+        tc = TrafficClass('delaunay', g, matrix)
+        ta = TrafficAssignment()
+        ta.set_classes([tc])
+        ta.set_time_field('distance')
+        ta.set_capacity_field('capacity')
+        ta.set_vdf('BPR')
+        ta.set_vdf_parameters({"alpha": 0, "beta": 1.0})
+        ta.set_algorithm('all-or-nothing')
+        ta.execute()
+
+        report = {"setup": str(ta.info())}
+        data = [result_name, "Delaunay assignment", self.procedure_id, str(report), ta.procedure_date, '']
+        conn.execute("""Insert into results(table_name, procedure, procedure_id, procedure_report, timestamp,
+                                            description) Values(?,?,?,?,?,?)""", data)
+        conn.commit()
+        conn.close()
+
+        cols = []
```

Comment:
```suggestion
        cols = [x for v in matrix.view_names for x in [f'{v}_ab', f'{v}_ba', f'{v}_tot']]
```

I do love me some list comps